packages/kbn-plugin-helpers/lib/utils.js

```diff
@@ -42,7 +42,7 @@ function resolveKibanaPath(path) {
 }
 
 function readFtrConfigFile(log, path, settingOverrides) {
-  return require(resolveKibanaPath('src/functional_test_runner')) // eslint-disable-line import/no-dynamic-require
+  return require(resolveKibanaPath('packages/kbn-test/src/functional_test_runner')) // eslint-disable-line import/no-dynamic-require
```

Comment:
We should be able to import this as `@kbn/test`, as long as `readConfigFile()` is exported from the root of the functional test runner.

Reply:
On it

Reply:
It does feel kinda odd to not declare the dependency for `@kbn/test`, in `@kbn/plugin-helpers`.